app/models/miq_ae_domain.rb

```diff
@@ -100,6 +100,17 @@ def editable_property?(property)
 
   alias editable_contents? editable?
 
+  def update_attributes(hash)
+    not_editable = hash.keys.reject { |p| editable_property?(p) }
```

Comment:
Suggest using `non_editable` for the variable name and `"Non  editable attributes..."` in the message text.